pkg/controller/netflow_test.go

```diff
@@ -124,24 +124,40 @@ func (cont *AciController) netflowPolUpdate(nfp *netflowpolicy.NetflowPolicy) ap
 	nf.SetAttr("dstAddr", nfp.Spec.FlowSamplingPolicy.DstAddr)
 	if nfp.Spec.FlowSamplingPolicy.Version == "netflow" {
 		nf.SetAttr("ver", "v5")
-	}
-	if nfp.Spec.FlowSamplingPolicy.Version == "ipfix" {
+	} else if nfp.Spec.FlowSamplingPolicy.Version == "ipfix" {
```

Comment:
I notice that this func is duplicating most of what handleNetflowPolUpdate does in netflow.go. Can you rework the code to exercise handleNetflowPolUpdate instead?

Reply:
Will work on that.